torch/csrc/distributed/rpc/init.cpp

```diff
@@ -67,6 +67,7 @@ PyObject* rpc_init(PyObject* /* unused */) {
               "local_value",
               &PyRRef::localValue,
```

Comment:
Just realized that these are all part of the public user API and we should have good documentation for all of these. We should define all of these methods in api.py and have proper python docs. Don't have to do this as part of this PR.